Codellama/README.md

```diff
@@ -0,0 +1,15 @@
+# Local Test of the Model
```

Comment:
You changed the name here, but I think it also shouldn't be the codellama directory?